client/src/components/UserPrompt.js

```diff
@@ -0,0 +1,54 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { MatchContext } from '../ContextProvider/match';
+import { useUserState } from '../ContextProvider/user';
+import { Button, Snackbar } from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+
+const initialRole = {
+  color: '',
+  title: '',
+  phase: '',
```

Comment:
Phase and title will always have corresponding values, so is there a way for you to only have to worry about one of them in state?

Reply:
Title is only used to make the prompt more personal. It can be removed but then the prompt will be generic like "it is your turn now".

Another way to keep prompt personal and removing title would be to use either if statements or switch to create an alert based on phase's value. The reason i did not do this because I thought the current way was simpler

Reply:
Added a function that uses a switch statement to return the correct text for the alert. Removed title as a state